nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -6,21 +6,19 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can use the Node Maintenance Operator to place nodes in maintenance mode. This is a standalone version of the Node Maintenance Operator that is independent of {VirtProductName} installation.
-
-[NOTE]
-====
-If you have installed {VirtProductName}, you must use the Node Maintenance Operator that is bundled with it.
-====
+You can use the Node Maintenance Operator to place nodes in maintenance mode, by using the `oc adm` utility, or `NodeMaintenance` custom resources (CRs).
 
 include::modules/eco-about-node-maintenance-standalone.adoc[leveloffset=+1]
 
-include::modules/eco-maintaining-bare-metal-nodes.adoc[leveloffset=+1]
-
 [id="installing-standalone-nmo"]
 == Installing the Node Maintenance Operator
 You can install the Node Maintenance Operator using the web console or the OpenShift CLI (`oc`).
 
+[NOTE]
+====
+If {VirtProductName} v4.10.z or less is installed in your cluster, it includes an outdated version of the Node Maintenance Operator version.
```

Comment:
Do they need to do anything about it in this case?

```suggestion
If {VirtProductName} version 4.10 or less is installed in your cluster, it includes an outdated version of the Node Maintenance Operator.
```

Reply:
Thanks @jeana-redhat , updated as suggested

Reply:
No, no action required. 

Feedback from SME earlier:
By looking in https://issues.redhat.com/browse/CNV-17623?focusedCommentId=20673382&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-20673382 I see that CNV suggest that a user can and should use "our" NMO before doing an upgrade if he/she want the node to stay under maintenance while doing the CNV upgrade...

I would suggest to leave the note, without an action from the user, as follows:

If OpenShift Virtualization v4.10.z or less is installed in your cluster, it includes an outdated version of the Node Maintenance Operator version.